.github/workflows/backport.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  pull_request_target:
+    branches:
+      - master
+    types:
+      - labeled
+      - closed
+
+jobs:
+  backport:
+    name: Backport PR
+    if: |
+      github.event.pull_request.merged == true
+      && contains(github.event.pull_request.labels.*.name, 'auto-backport')
+      && (
+        (github.event.action == 'labeled' && github.event.label.name == 'auto-backport')
+        || (github.event.action == 'closed')
+      )
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+        with:
+          repository: 'elastic/kibana-github-actions'
+          ref: main
+          path: ./actions
+
+      - name: Install Actions
+        run: npm install --production --prefix ./actions
+
+      - name: Run Backport
+        uses: ./actions/backport
+        with:
+          github_token: ${{secrets.BACKPORT_ACTION_TOKEN}}
+          commit_user: kibanamachine
```

Comment:
Do you have a Github user for beats similar to [kibanamachine](https://github.com/kibanamachine)?

Reply:
Yeah, they should use something other than `kibanamachine`, probably `elasticmachine` or their `jenkins-beats-ci` bot?